README.md

```diff
@@ -138,6 +128,32 @@ loader.load(
 );
 ```
 
+### Use with WebGPURenderer
+
+Starting from v3, we provide [WebGPURenderer](https://github.com/mrdoob/three.js/blob/master/examples/jsm/renderers/webgpu/WebGPURenderer.js) compatibility.
+To use three-vrm with WebGPURenderer, specify the WebGPU-compatible `MToonNodeMaterialLoaderPlugin` for the `mtoonMaterialPlugin` option of `VRMLoaderPlugin`.
+
+```js
+import { VRMLoaderPlugin, MToonNodeMaterialLoaderPlugin } from '@pixiv/three-vrm';
+
+// ...
+
+// Register a VRMLoaderPlugin
+loader.register((parser) => {
+
+  // create a WebGPU compatible MToon loader plugin
+  const mtoonMaterialPlugin = new MToonNodeMaterialLoaderPlugin(parser);
```

Comment:
```suggestion
  const mtoonMaterialPlugin = renderer.isWebGPURenderer === true
    ? new MToonNodeMaterialLoaderPlugin(parser)
    : undefined; // fallbacks to the default `MToonMaterialLoaderPlugin`
```

Reply:
This is kinda redundant considering that the document already says that it's for `WebGPURenderer`.